AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -436,6 +436,7 @@ comnuan.com#%#//scriptlet("set-constant", "cmnnrunads", "true")
 marriedgames.com.br#%#//scriptlet("set-constant", "adning_no_adblock", "true")
 parasportontario.ca#%#//scriptlet("abort-on-property-read", "closeBlockerModal")
 ||studopedia.ru/ggl/$script
+estadao.com.br##.fc-ab-root
```

Comment:
We have rules for it https://github.com/AdguardTeam/AdguardFilters/blob/10a8da3c87ef4b9239f9cfbfdb67baf73d9c84ac/AnnoyancesFilter/sections/antiadblock.txt#L391

Reply:
but I cannot reproduce both elements

Reply:
Also not reproduced on my end.

Reply:
I lost the element, removed the rule. Cookie banner is still present ( espanol IPs):

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/109940428-1cf26480-7ce3-11eb-9ec2-40988d04c603.png)
</details><br/>